client/my-sites/checkout/composite-checkout/components/item-variation-picker/item-variation-dropdown.tsx

```diff
@@ -95,178 +14,38 @@ export const ItemVariationDropDown: FunctionComponent< ItemVariationPickerProps
 } ) => {
 	const translate = useTranslate();
 
-	const [ open, setOpen ] = useState( false );
```

Comment:
All the selector logic and keyboard behaviour is handled by `CustomSelectControl` internally.